### PR TITLE
Aleksei/fix subscribe to more

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

subscribeToMore should contain document instead of query

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
I just tested it and it is working great. Even with the "outlier" `cosmos1u6d6usvgz8hjzd7dwgpfegk4xf9j83frc7wunf`. Awesome job 👍

File: src/components/wallet/PageTransactions.vue [hunk 1]
```diff
@@ -162,6 +161,26 @@ export default {
...
+              networkId: this.network,
+              address: this.address,
+              pageNumber: this.pageNumber
+            },
+            // Transform the previous result with new data
+            updateQuery: (previousResult, { fetchMoreResult }) => {
+              this.lastLoadedRecordsCount =
+                fetchMoreResult.transactionsV2.length
+              return {
+                __typename: previousResult.transactionsV2.__typename,
```
Comment (faboweb) [inline src/components/wallet/PageTransactions.vue]: this is needed?

**Comment by faboweb** [comment]
unit tests are failing

File: src/components/wallet/PageTransactions.vue (hunk 1 above)
Comment (iambeone) [reply src/components/wallet/PageTransactions.vue]: Which line? __typename? Yap

File: tests/unit/specs/components/wallet/PageTransactions.spec.js [hunk 2]
```diff
@@ -216,6 +220,8 @@ describe(`PageTransactions`, () => {
   })
 
   it(`should load more transactions on loadMore action`, async () => {
+    $store.getters.address = "address"
+    $store.getters.network = "cosmos-hub"
```
Comment (Bitcoinera) [inline tests/unit/specs/components/wallet/PageTransactions.spec.js]: Wouldn't it be better to add the network getter in the beforeEach?

File: .vscode/settings.json [hunk 3]
```diff
@@ -13,5 +13,7 @@
   "css.lint.important": "warning",
   "css.lint.universalSelector": "warning",
   "css.lint.unknownVendorSpecificProperties": "warning",
-  "css.lint.zeroUnits": "warning"
+  "css.lint.zeroUnits": "warning",
+  "liveServer.settings.root": "/dist",
+  "liveServer.settings.wait": 100000
```
Comment (Bitcoinera) [inline .vscode/settings.json]: I don't think this is part of the PR 😅

File: src/components/wallet/PageTransactions.vue (hunk 1 above)
Comment (faboweb) [reply src/components/wallet/PageTransactions.vue]: why?

Comment (Bitcoinera) [reply src/components/wallet/PageTransactions.vue]: I tested it and it works without it. If it is needed for some reason that I cannot see right now we can add it later.

Let's merge this! 💪

Comment (iambeone) [reply src/components/wallet/PageTransactions.vue]: https://vue-apollo.netlify.com/guide/apollo/pagination.html

Comment (Bitcoinera) [reply src/components/wallet/PageTransactions.vue]: But it is working well... 🤷‍♀

**Pull request merged**